fri/src/verifier.rs

```diff
@@ -154,18 +143,13 @@ fn verify_query<FC: FriConfig>(
                 &[evals.clone()],
                 &step.opening_proof,
             )
-            .map_err(|e| VerificationError::CommitPhaseMmcsError(e))?;
+            .map_err(VerificationError::CommitPhaseMmcsError)?;
 
         let mut xs = [x; 2];
         xs[index_sibling % 2] *= FC::Challenge::two_adic_generator(1);
         // interpolate and evaluate at beta
         folded_eval = evals[0] + (beta - xs[0]) * (evals[1] - evals[0]) / (xs[1] - xs[0]);
 
-        for o in &smaller_openings[log_height - 1] {
```

Comment:
This doesn't work, I think it's because with this wonky indexing scheme, the smaller matrices should be folded in by masking the top bits, instead of the bottom bits as MMCS currently does.